### ISSUE TITLE
[MU4 Issue] Instruments>Add staff: The position of added staff is not updated properly if that staff is added not to the bottom 

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#12131]
**Describe the bug**
https://github.com/musescore/MuseScore/pull/10526#issuecomment-1161726648

The position of added staff is not updated properly if added to the top/middle staff. It's updated after reopening the score (see video) 

https://user-images.githubusercontent.com/90187801/175040104-6fc8127c-e3f0-43f4-b3bb-127b6503d0f5.mp4

**To Reproduce**
1. Create a piano instruments 
2. In Instruments panel add staff to the top staff (Treble clef)

**Platform information**
macOS, Windows, Linux

**Additional context**
#10526

**Comment by wizofaus** [comment]
Weird, for me it's always added to the bottom in both the score and the instruments panel, which actually seems OK, there's not that many times you'd want to add a staff between two existing staves, and there's no way I can see you could indicate you wanted it added above the existing staves (e.g. an ossia staff)

**Comment by DmitryArefiev** [comment]
> Weird, for me it's always added to the bottom in both the score and the instruments panel, which actually seems OK, there's not that many times you'd want to add a staff between two existing staves, and there's no way I can see you could indicate you wanted it added above the existing staves (e.g. an ossia staff)

The current bug is not in master now because #10526 is not merged yet..

**Comment by wizofaus** [comment]
So if this bug is only happening for a particular PR build, does it make sense to leave open? What actually is the expected behaviour here?

**Comment by DmitryArefiev** [comment]
#10526 should be in master soon (after code review) so I propose to leave the bug in Open state

**Comment by cbjeukendrup** [comment]
The behaviour of this issue has changed: since https://github.com/musescore/MuseScore/pull/12293, staves are always added at the bottom, so the instruments panel is now in sync with the score.

@DmitryArefiev Do you think we should close this issue, or update it, or create a new task for inserting staves after the selected staff?

**Comment by DmitryArefiev** [comment]
@cbjeukendrup Yes, let's close it. The initial bug (position is not updated properly) doesn't occur anymore
